slideshow/slideshow_test.go

```diff
@@ -0,0 +1,79 @@
+package slideshow
+
+import (
+	"fmt"
+	"testing"
+)
+
+func TestReadSlideshow(t *testing.T) {
+	templateName := "../TestInput/test.slideshow"
+
+	slideshow := NewSlideshow(templateName)
+
+	expectedImages := []string{"../TestInput/Jn01.1-18-title.jpg", "../TestInput/./VB-John 1v1.jpg", "../TestInput/./VB-John 1v3.jpg", "../TestInput/./VB-John 1v4.jpg", "../TestInput/./VB-John 1v5a.jpg",
+		"../TestInput/./VB-John 1v5b.jpg", "../TestInput/./VB-John 1v6.jpg", "../TestInput/Gospel of John-credits.jpg"}
```

Comment:
test.slideshow doesn't have ../TestInput, but when we parse the slideshow file, we append the current path directory to it so that it is relative to the executable, not the slideshow file.

Reply:
Can you give me the expected value on your machine @Davideo37 ?

Reply:
I don't have access to my laptop to test it again but I believe the expected value was the same thing with the "../TestInput/./image-name.jpg". However the string being parsed and compared was just "./image-name.jpg", so it was failing